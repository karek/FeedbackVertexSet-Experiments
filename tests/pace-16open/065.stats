66,127,5
