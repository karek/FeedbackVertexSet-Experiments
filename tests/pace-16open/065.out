21
21 32 35 39 22 30 5 25 17 8 31 1 44 4 29 13 51 18 57 9 11 
